src/Ombi.Core/Helpers/TvShowRequestBuilder.cs

```diff
@@ -16,23 +18,35 @@ namespace Ombi.Core.Helpers
     public class TvShowRequestBuilder
     {
 
-        public TvShowRequestBuilder(ITvMazeApi tvApi)
+        public TvShowRequestBuilder(ITvMazeApi tvApi, IMovieDbApi movApi)
         {
             TvApi = tvApi;
+            MovieDbApi = movApi;
         }
         
         private ITvMazeApi TvApi { get; }
+        private IMovieDbApi MovieDbApi { get; }
 
         public ChildRequests ChildRequest { get; set; }
         public List<SeasonsViewModel> TvRequests { get; protected set; }
         public string PosterPath { get; protected set; }
         public DateTime FirstAir { get; protected set; }
         public TvRequests NewRequest { get; protected set; }
         protected TvMazeShow ShowInfo { get; set; }
+        protected List<TvSearchResult> Result { get; set; }
 
         public async Task<TvShowRequestBuilder> GetShowInfo(int id)
         {
             ShowInfo = await TvApi.ShowLookupByTheTvDbId(id);
+            Result = await MovieDbApi.SearchTv(ShowInfo.name);
+            foreach (TvSearchResult result in Result) {
+                if (result.Name == ShowInfo.name)
+                {                  
+                    var showIds = await MovieDbApi.GetTvExternals(result.Id);
+                    ShowInfo.externals.imdb = showIds.imdb_id;
```

Comment:
Since we are here we can store `TheMovieDbId` too.

Reply:
`ShowInfo` here is of type `TvMazeShow` and it does not have `TheMovieDbId` as one of its attributes. Shall I extend the type to include this and then I can set it here?

Reply:
Wrote a DB migration and extended the TvRequests to include `TheMovieDbId` and it seems to be working beautifully. Is this ok to do? If so I will push the commit.